charts/presto/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: presto
+description: The official Helm chart for Presto
+type: application
+version: 0.1.0
+appVersion: "0.278"
```

Comment:
how will this version be kept up to date as we create new Presto releases?

Reply:
The source code with the value will be updated by a PR (created manually or automatically via CI script).

Please consider that the value of `appVersion` is the default tag for docker image to be used and can be overwritten by a user. So currently `appVersion` means that the chart deploys `ahanaio/prestodb:$appVersion` by default.
Usually application and helm chart releases happen separately. It's not needed to release helm chart immediately with the application being released. For example, Apache Airflow releases their helm chart only once for minor application version, i.e chart `airflow:1.7.0` uses `airflow:2.4.1` as the default version, but it's possible to try and use `airflow:2.5.0`.

Not sure it answers you question, please ask for more details if needed.

Reply:
Also it might be a good idea to create separate git repo for the helm chart, because of different release process and test suites.